src/app/view/components/_common/dot-workflows-actions-selector-field/services/dot-workflows-actions.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { CoreWebService } from 'dotcms-js';
+import { pluck } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { RequestMethod } from '@angular/http';
+
+export interface DotWorkflowsAction {
+    assignable: boolean;
+    commentable: boolean;
+    condition: string;
+    icon: string;
+    id: string;
+    name: string;
+    nextAssign: string;
+    nextStep: string;
+    nextStepCurrentStep: boolean;
+    order: number;
+    owner?: string;
+    roleHierarchyForAssign: boolean;
+    schemeId: string;
+    showOn: string[];
+}
```

Comment:
should it go in its own file?

Reply:
In fact, already existed a interface for this.